recipes/xrandr.lua

```diff
@@ -131,7 +130,6 @@ local function xrandr()
                               function()
                                  state.timer:stop()
                                  state.timer = nil
-                                 state.iterator = nil
```

Comment:
Shouldn't this set `state.index` to `nil` so that the thing also works if I let it apply something and then call it again?